### PR TITLE
Add Scaleway object storage provider

### PR DESCRIPTION
**Pull request by angristan**
I added this one based on the others.

Tests are passing:

```
devenv :: ~/juicesync/object ‹master*› » SCW_TEST_BUCKET=juicetest.s3.fr-par.scw.cloud go test -run TestScw
1 
PASS
ok      github.com/juicedata/juicesync/object   4.698s
```

### CODE REVIEW

**Comment by angristan** [comment]
Can confirm it works with `juicefs` too 👍

**Review by davies — APPROVED** [review]

**Comment by davies** [comment]
@angristan Thanks! Do you mind update the readme to mention this?

**Comment by angristan** [comment]
I added a line in the README 🙂

**Review by nlm — CHANGES_REQUESTED** [review]
Nice work @angristan!
I think you should also add something like this in the notes section:
```
- SCW:
  * The credential can be provided by environment variable `SCW_ACCESS_KEY` and `SCW_SECRET_KEY` .
```

**Comment by davies** [comment]
@nlm I only saw `SCW_ACCESS_KEY ` in test case, did I missed something?

File: object/scw.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	bucket := hostParts[0]
+	region := hostParts[2]
+	endpoint = uri.Host[len(bucket)+1:]
+
+	if accessKey == "" {
+		accessKey = os.Getenv("SCW_ACCESS_KEY")
+	}
+	if secretKey == "" {
+		secretKey = os.Getenv("SCW_SECRET_KEY")
+	}
```
Comment (angristan) [inline object/scw.go:60]: I added this, like in the `OBS` provider. I also added the corresponding line in the readme.

**Pull request merged**